content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -178,28 +233,22 @@ Detailed network timing data for the loading of an application’s resources are
 | `resource.provider.type`      | string | The resource provider type (for example `first-party`, `cdn`, `ad`, `analytics`).                                            |
 
 
-[1]: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest
-[2]: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
+[1]: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
 {{% /tab %}}
 {{% tab "Long Task" %}}
 
-[Long tasks][1] are tasks that block the main thread for 50 milliseconds or more. They may cause high input latency, delayed time to interaction, etc. Understand what causes these long tasks in your browser performance profiler.
-
 ### Long task timing metrics
 
 | Metric  | Type   | Description                |
 |------------|--------|----------------------------|
 | `long_task.duration` | number | Duration of the long task. |
 
 
-[1]: https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API
 {{% /tab %}}
 {{% tab "Error" %}}
 
-Front-end errors are collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
-
-### Error origins
-Front-end errors are split in 4 different categories depending on their `error.origin`:
+### Error sources
+Front-end errors are split in 4 different categories depending on their `error.source`:
 
 - **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the documentation][1].
```

Comment:
Can we remove the line "Specific attributes...documentation" -> we're linking to the top of the doc which is unhelpful, and we're mentioning network error attributes anyway right after this section.